src/filesystem/nodes/node_data/mod.rs

```diff
@@ -240,6 +241,18 @@ impl NodeData {
         })
     }
 
+    pub(crate) fn empty_file() -> Self {
+        Self::File {
+            permissions: FilePermissions::default(),
+            associated_data: HashMap::new(),
+            content: FileContent::Encrypted {
+                locked_access_key: SymLockedAccessKey::default(),
```

Comment:
This is definitely a no-no, we'll attempt to decrypt it and fail horribly even with a byte size of zero. I see the need for it to construct this type, but because it is a necessary field we probably need to create another variant for representing empty non-stub files regardless of whether they're encrypted or not.